aurora/main.f90

```diff
@@ -51,8 +49,12 @@ subroutine run(  &
   !                    This must be given for each charge state and time.
   !     par_loss_rates  real*8 (ir,nt)
   !                    Frequency for parallel loss on radial and time grids [1/s]
-  !     src_rad_prof real*8 (ir,nt)
-  !                    Radial profile of neutrals over time: n0(ir,t) = flx*src_rad_prof(ir,t).
+  !     source_2d real*8 (ir,nt)
+  !                    Radial profile of neutrals over time.
```

Comment:
Have a look at how `source_2d` (or its individual time slices, called `src_prof` in impden.f90) is used here:
https://github.com/fsciortino/Aurora/blob/6fc8f3c82ebe39a1c030d3177916e42cdad45d54/aurora/impden.f90#L321
and here:
https://github.com/fsciortino/Aurora/blob/6fc8f3c82ebe39a1c030d3177916e42cdad45d54/aurora/impden.f90#L328

From these lines you see that the source radial profile is used to set the first array index of the impurity densities, i.e. the neutral stage. This is quite counter-intuitive -- maybe even wrong, one may say -- because sometimes we impose these source radial profiles by describing them as an ionization profile of neutrals.... but in fact, they are taken to represent the neutral stage themselves, rather than the first ionization stage. 

Looking at the lines above, I realized that the physical units of `source_2d` should be `particles`, not `particles/cm/s` or `particles/cm`. This is because each element of this array corresponds to the number of particles at each location and at each time. Would you agree??